deploy_hpo.sh

```diff
@@ -50,8 +51,15 @@ function usage() {
 	echo " -o: build with specific hpo container image name [Default - kruize/hpo:<version>]"
 	echo " -n: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
 	echo " -d: Config maps directory [Default - manifests/configmaps]"
+	echo " Environment Variables to be set: REG_EMAIL, REG_UNAME, REG_PASS"
 	exit -1
 }
+# check if registry credentials are set as Environment Variables
+if [ -z $REG_UNAME ] || [ -z $REG_PASS ] || [ -z $REG_EMAIL ]; then
```

Comment:
This check should be made only if the cluster type is openshift or minikube

Reply:
Seems to be used only in openshift-helpers script, so the PR checks should work fine on other cluster types, if condition is added to check cluster type as openshift

Reply:
Added condition for both Minikube and Openshift.